test/io/test_io.py

```diff
@@ -883,17 +883,19 @@ def test_admin_live_dependent_on_main_app(defaultenv):
         response = postgrest.admin.get("/live")
         assert response.status_code == 503
 
+
 @pytest.mark.parametrize("specialhostvalue", FIXTURES["specialhostvalues"])
 def test_admin_works_with_host_special_values(specialhostvalue, defaultenv):
     "Should get a success from the admin live and ready endpoints when using special host values for the main app"
 
     with run(env=defaultenv, port=freeport(), host=specialhostvalue) as postgrest:
 
-      response = postgrest.admin.get("/live")
-      assert response.status_code == 200
+        response = postgrest.admin.get("/live")
+        assert response.status_code == 200
+
+        response = postgrest.admin.get("/ready")
+        assert response.status_code == 200
```

Comment:
I'm not sure what happened here. I guess you ran `postgrest-style`? I just don't understand why that code was formatted like that. It should have stopped CI from passing and never been committed like that.

Reply:
🤷  @steve-chavez ran the linter and commited that

Reply:
This one is mine actually, I ran `postgrest-style` and commited. Not sure what happened too, but I assumed no harm is done if it's added here.

Reply:
Hm. It's certainly worth more investigation. As I said, I would have expected CI to pick that up and fail in main already. It doesn't. What else is CI missing to notice right now? :confused: